Cargo.toml

```
@@ -180,7 +180,7 @@ dir-diff = "0.3.2"
 dirs-next = "2.0.0"
 dlopen2 = "0.5.0"
 eager = "0.1.0"
-ed25519-dalek = "=1.0.1"
+ed25519-dalek = "2.0.0"
 ed25519-dalek-bip32 = "0.2.0"
```

```suggestion
ed25519-dalek-bip32 = "0.3.0"
```
fyi `ed25519-dalek-bip32` just bumped out 0.3.0 release aligned to new API as well :partying_face: